src/int.rs

```diff
@@ -170,89 +170,104 @@ impl I256 {
     }
 
     /// Cast to a primitive `i8`.
+    #[inline]
```

Comment:
Any reason why these ones weren't also marked as `#[inline(always)]`?

Reply:
Nope. You can change it if you think that's best.